models/nc-news.models.js

```diff
@@ -53,7 +58,7 @@ exports.fetchArticles = (articleId, topicQuery) => {
       }
 
       queryStr += ` GROUP BY articles.article_id 
-      ORDER BY created_at DESC;`;
+      ORDER BY ${sortBy} ${orderBy};`;
```

Comment:
You need to sanitise these queries before they're allowed to be put into the SQL query - i.e. set up which are allowed sort_by and order_by queries. 
At the moment, this would allow them to replace the sortBy/orderBy with their own SQL query.